lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -186,7 +202,15 @@ export class CryptoOps implements ICrypto {
         const signatureBuffer = await this.browserCrypto.sign(cachedKeyPair.privateKey, tokenBuffer);
         const encodedSignature = this.b64Encode.urlEncodeArr(new Uint8Array(signatureBuffer));
 
-        return `${tokenString}.${encodedSignature}`;
+        const signedJwt = `${tokenString}.${encodedSignature}`;
+
+        if (signJwtMeasurement) {
+            signJwtMeasurement.endMeasurement({
+                success: true
+            });
+        }
+
```

Comment:
Do we need an else block here ?

Reply:
Nope, don't need to do anything if there isnt a performance client.